l10n_it_delivery_note/models/account_invoice.py

```diff
@@ -141,6 +141,16 @@ def update_delivery_note_lines(self):
                     for note_line in dn.line_ids.filtered(
                         lambda l: l.invoice_status == DOMAIN_INVOICE_STATUSES[2]
                     ):
+                        if not dn_invoice_lines:
+                            for new_line in new_lines:
+                                if new_line[2]["sequence"] == (sequence - 1):
+                                    new_line[2]["name"] = _(
+                                        """{} and "{}" of {}"""
+                                    ).format(
+                                        new_line[2]["name"],
+                                        dn.name,
+                                        dn.date.strftime(DATE_FORMAT),
+                                    )
```

Comment:
**suggestion**: using the `.format()` method is discouraged because it can be source of vulnerabilities.
See [here](https://github.com/OCA/pylint-odoo/issues/302#issue-758472967) the comment from Yajo

Reply:
Meglio fare il link direttamente a CONTRIBUTING:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#idioms

>Prefer `%` over `.format()`, prefer `%(varname)` instead of positional.
  This is better for translation
  `and security <https://github.com/OCA/pylint-odoo/issues/302#issue-758472967>`__.

Reply:
Ho corretto, grazie